.taskcluster.yml

```diff
@@ -256,7 +256,7 @@ tasks:
                                             ~/.local/bin/taskgraph action-callback
                                         else: >
                                             PIP_IGNORE_INSTALLED=0 pip install --user /builds/worker/checkouts/taskgraph &&
-                                            PIP_IGNORE_INSTALLED=0 pip install --user arrow taskcluster pyyaml &&
+                                            PIP_IGNORE_INSTALLED=0 pip install --user chunkify &&
```

Comment:
Looks like dependency not used and this one added: https://github.com/mozilla-mobile/firefox-ios/pull/6459/files#diff-a728f7e52d751b98eafa856e45594533339b44f229d7b83f930df335391e7e15L259